doc/release/1.3.0-notes.rst

```diff
@@ -24,41 +24,578 @@ For running on PyPy, PyPy3 6.0+ and NumPy 1.15.0 are required.
 Highlights of this release
 --------------------------
 
--
+- Three new ``stats`` functions, a rewrite of ``pearsonr``, and an exact
+  computation of the Kolmogorov-Smirnov two-sample test
+- A new Cython API for bounded scalar-function root-finders in `scipy.optimize`
+- Substantial ``CSR`` and ``CSC`` sparse matrix indexing performance
+  improvements
+- Added support for interpolation of rotations with continuous angular
+  rate and acceleration in ``RotationSpline``
+
 
 New features
 ============
 
+`scipy.interpolate` improvements
+--------------------------------
+
+New class ``CubicHermiteSpline`` is introduced. It is a piecewise-cubic 
+interpolator which matches observed values and first derivatives. Existing 
+cubic interpolators ``CubicSpline``, ``PchipInterpolator`` and 
+``Akima1DInterpolator`` were made subclasses of ``CubicHermiteSpline``.
+
+`scipy.io` improvements
+-----------------------
+
+`scipy.io.arff.loadarff` now supports relational attributes
+
+`scipy.io.mmread` can now parse Matrix Market format files with empty lines
+
+`scipy.linalg` improvements
+---------------------------
+
+Added wrappers for ``?syconv`` routines, which convert a symmetric matrix 
+given by a triangular matrix factorization into two matrices and vice versa.
+
+`scipy.linalg.clarkson_woodruff_transform` now uses an algorithm that leverages
+sparsity. This may provide a 60-90 percent speedup for dense input matrices.
+Truly sparse input matrices should also benefit from the improved sketch
+algorithm, which now correctly runs in ``O(nnz(A))`` time.
+
+Added new functions to calculate symmetric Fiedler matrices and
+Fiedler companion matrices, named `scipy.linalg.fiedler` and 
+`scipy.linalg.fiedler_companion`, respectively. These may be quite useful
+for root finding.
+
+`scipy.ndimage` improvements
+----------------------------
+
+``_gaussian_kernel1d`` performance may improve by an order of magnitude in
```

Comment:
this needs to refer to a public function

Reply:
I've adjusted based on a quick inspection of source code that calls this private function.